components/dictionary.txt

```diff
@@ -1,6 +1,7 @@
 actin
```

Comment:
Some of these can probably come out now that we've switched the dataset but not a big deal.